src/CollectionTrait.php

```diff
@@ -40,7 +40,7 @@ public function distinct()
     /**
      * Returns a lazy collection with items from all $collections passed as argument appended together
      *
-     * @param \Traversable[]|array ...$collections
+     * @param (array|\Traversable)[] ...$collections
```

Comment:
Sure, I'll change it to the suggested format.

According to the upcoming PSR-5 I think `(array|Traversable)[]` is the format that should be used in this case (see: [ABNF](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#abnf) and [Arrays#3](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#arrays)). But I agree that the format `array[]|Traversable[]` seems to be more used in the wild for now.